examples/config.yaml

```diff
@@ -42,6 +42,12 @@ providers:
     password: 'password'
   pushbullet:
     access_token: 'o.AbCdEfGhIjKlMnOpQrStUvWxYz012345'
+  nowsms:
+    username: 'username'
+    password: 'password'
+    phone_numbers:
```

Comment:
You should specify the phone numbers that will receive the SMS on the `To` attribute of the receivers. Like this https://github.com/messagebird/sachet/blob/master/examples/config.yaml#L52. 

And not have another configuration option. This is more consistent and eases the move from one privider to another